docs/EmbeddedAppIOS.md

```diff
@@ -15,7 +15,9 @@ Since React makes no assumptions about the rest of your technology stack – it
 - [Node.js](http://nodejs.org)
   - Install **nvm** with [its setup instructions here](https://github.com/creationix/nvm#installation). Then run `nvm install node && nvm alias default node`, which installs the latest version of Node.js and sets up your terminal so you can run it by typing `node`.  With nvm you can install multiple versions of Node.js and easily switch between them.
   - If you are using Node 5.0 or newer, we recommend installing npm 2, which is much faster than npm 3. After installing Node, run `npm install -g npm@2`
-- Install your copy of React Native under your `node_modules` directory where your JS resides.
+- Install the react-native node	package in a subdirectory of your project.  It will be helpful, when we're bundling and serving js files with the React Native development server (see below), to have this in a directory by itself, with no other child or sibling js files.  We suggest creating a `NodeModulesParent` directory under the root directory of your project.  Then:
```

Comment:
Not exactly sure what tab/double space you mean.
There was a weird character between cd and NodeModulesParent, removed that.

Why NodeModulesParent:
Removed it.  I spent a long time thrashing because the server would be
sucking in extra js files.  In the midst of thrashing, for the sake of
overkill, just wanted to put node_modules somewhere completely alone, with
nothing else even as a sibling.




On Fri, Dec 18, 2015 at 3:17 AM, Satyajit Sahoo <notifications@github.com>
wrote:

> In docs/EmbeddedAppIOS.md
> <https://github.com/facebook/react-native/pull/4828#discussion_r48015996>:
>
> > @@ -15,7 +15,9 @@ Since React makes no assumptions about the rest of your technology stack – it
> >  - [Node.js](http://nodejs.org)
> >    - Install **nvm** with [its setup instructions here](https://github.com/creationix/nvm#installation). Then run `nvm install node && nvm alias default node`, which installs the latest version of Node.js and sets up your terminal so you can run it by typing `node`.  With nvm you can install multiple versions of Node.js and easily switch between them.
> >    - If you are using Node 5.0 or newer, we recommend installing npm 2, which is much faster than npm 3. After installing Node, run `npm install -g npm@2`
> > -- Install your copy of React Native under your `node_modules` directory where your JS resides.
> > +- Install the react-native node	package in a subdirectory of your project.  It will be helpful, when we're bundling and serving js files with the React Native development server (see below), to have this in a directory by itself, with no other child or sibling js files.  We suggest creating a `NodeModulesParent` directory under the root directory of your project.  Then:
>
> Remove extra tab and double spaces here :)
>
> BTW why have a special NodeModulesParent folder? NPM modules are installed
> under a node_modules directory itself.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/react-native/pull/4828/files#r48015996>.
>